raft/src/main/java/org/apache/kafka/raft/KafkaRaftClient.java

```diff
@@ -3337,6 +3433,29 @@ private RequestSendResult maybeSendUpdateVoterRequest(FollowerState state, long
         );
     }
 
+    private RequestSendResult maybeSendAddVoterRequest(
+        FollowerState state,
+        long currentTimeMs
+    ) {
+        return maybeSendRequest(
+            currentTimeMs,
+            state.leaderNode(channel.listenerName()),
+            this::buildAddVoterRequest
+        );
+    }
+
+    private RequestSendResult maybeSendRemoveVoterRequest(
+        FollowerState state,
+        long currentTimeMs,
+        ReplicaKey replicaKey
```

Comment:
Flip the order of these parameters. The kraft module has a pattern of using the last parameter as the current time when needed.